warehouse/query-core/src/main/java/datawave/query/config/RemoteQueryConfiguration.java

```diff
@@ -89,6 +90,17 @@ public static RemoteQueryConfiguration create(RemoteEventQueryLogic remoteQueryL
         return create(remoteQueryLogic.getConfig());
     }
 
+    /**
+     * Factory method that creates a RemoteQueryConfiguration deep copy from a RemoteQueryLogic
+     *
+     * @param remoteQueryLogic
+     *            - a configured RemoteQueryLogic
+     * @return - a RemoteQueryConfiguration
+     */
+    public static RemoteQueryConfiguration create(RemoteEdgeQueryLogic remoteQueryLogic) {
+        return create(remoteQueryLogic.getConfig());
+    }
```

Comment:
Could you use the `RemoteQueryLogic` interface as the parameter type for this and thus have a single `create` method that applies to both the `RemoteEdgeQueryLogic` and the `RemoveEventQueryLogic`? You would have to add `getConfig()` to the RemoteQueryLogic interface though...

Reply:
Thank you for reviewing.  That seemed like a simple enough add that it was not worth it.  I may remove those convience methods altogether.